### PR TITLE
Soarian FHIR R4 DiagnosticReport Resource

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
----
Review Request for Soarian FHIR R4 DiagnosticReport Resource
PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
DrxReport_1
DrxReport_2
DrxReport_3
DrxReport_4
DrxReport_5
DrxReport_6
DrxReport_7

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by HRNWriter — CHANGES_REQUESTED** [review]
Please see my comments.

PRODOC-160857.

File: layouts/soarian/r4/clinical/diagnostics.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a diagnosis is confirmed, it would be recorded as a Condition so that it is part of the historical patient record.</p>
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply layouts/soarian/r4/clinical/diagnostics.html]: Changed "Note," to "Note:"

File: layouts/soarian/r4/clinical/diagnostics.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+<p>The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical laboratory data and vital signs while the DiagnosticReport resource typically provides a textual set of information and interpretation after performing a diagnostic service such as a Radiology report or Pathology report.</p>
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply layouts/soarian/r4/clinical/diagnostics.html]: made changes as suggested

File: content/soarian/r4/clinical/diagnostics/diagnostic-report.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+{:toc}
+
+## Overview
+
+The DiagnosticReport resource includes information that is typically provided by a diagnostic service when investigations are complete. The information includes laboratory, microbiology, results reporting, and clinical notes exchange. Clinical notes include the three categories of Cardiology, Pathology, and Radiology.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
+The search results include the following fields if they contain values:
+
```
Comment (Aish1997) [inline content/soarian/r4/clinical/diagnostics/diagnostic-report.md:17]: How about the DiagnosticReport.identifier field?

Comment (DeepakGupta93) [reply content/soarian/r4/clinical/diagnostics/diagnostic-report.md:17]: We do not support DrxReport.identifier  field, we have identifier code and abbreviation in code section
User Story:
image
Documentation Word file:
image

File: content/soarian/r4/clinical/diagnostics/diagnostic-report.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example Search by Patient
+
+#### Request
+
+	GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/DiagnosticReport?patient=A879904FD2FE4B2D90C89FDA84E1285F
```
Comment (Aish1997) [inline content/soarian/r4/clinical/diagnostics/diagnostic-report.md:75]: We should probably be giving the public sandbox URL I suppose, could you please cross check and implement in all other instances as well

Comment (Aish1997) [reply content/soarian/r4/clinical/diagnostics/diagnostic-report.md:75]: You can update the public sandbox URL once implemented, will approve the PR

**Comment by DeepakGupta93** [comment]
> Add a comma (,) after "such as clinical laboratory data and vital signs" PRODOC-169058.

this line was updated as part of a commit before and split into 2 different lines

**Review by slvsridhar — APPROVED** [review]

**Review by vf056083 — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]

**Review by HRNWriter — APPROVED** [review]

**Pull request merged**